miniapps/meshing/mesh-minimal-surface.cpp

```diff
@@ -0,0 +1,1283 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
```

Comment:
I suggest renaming this to simply `minimal-surface`.

Reply:
`done`